.env.template

```diff
@@ -7,3 +7,9 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY=''
```

Comment:
Really minor, but I'd be tempted to just put `TRELLO_API_KEY=` here, and skip the empty quotes. The reason is a bit unnecessarily forward thinking, but:
* With the quotes there, as a user I might be tempted to put my variable inside quotes, e.g. `TRELLO_API_KEY='<my_key>'`
* And that will work fine right now, Flask doesn't care if there are quotes or not
* But Docker has other opinions, and can introduce a very subtle behaviour where it assumes the `'` are part of the actual variable, which is often annyoing to debug!